README.md

```diff
@@ -22,6 +22,7 @@ continuous development steps.
 ## Set a value
 
 `toml set --toml-path pyproject.toml tool.poetry.version 0.2.0`
+`toml set --toml-path pyproject.toml tool.poetry.authors[0] "Marc Rijken <marc@rijken.org>"`
```

Comment:
Is it possibel to extend this to add a value to the list? ie by using the next index?

Reply:
```suggestion
`toml set --toml-path pyproject.toml tool.poetry.authors[0] "Marc Rijken <marc@rijken.org>"`

When the index exists, the item is changed.  Otherwise, the item will be added to the list.
```